control-plane/subcommand/server-acl-init/command.go

```diff
@@ -139,12 +139,11 @@ func (c *Command) init() {
 	var unused bool
 	c.flags.BoolVar(&unused, "create-inject-namespace-token", false,
 		"Toggle for creating a connect injector token. Only required when namespaces are enabled. "+
```

Comment:
should we remove this flag? I think it was added for backward compatibility but at this point we don't support different versions of consul-k8s-control-plane and the helm chart